src/GraphQL.Analyzers/AnalyzerReleases.Shipped.md

```diff
@@ -0,0 +1,3 @@
+﻿; Shipped analyzer releases
+; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
```

Comment:
I'm looking at the docs referenced here, and I'm not sure that these comments are supposed to remain here, but rather are just placeholder text when auto-creating these files.  Do you know?  It would certainly be preferred if they remained (certainly in some fashion), but they are not markdown comments, as far as I can tell.

Reply:
This was added automatically when these files were created (also automatically). I think we can remove them because the built-in analyzer detects any change to analyzer descriptors and provides a fix to update these files.

Reply:
Tried to change it to markdown but got this error

> Analyzer release file 'AnalyzerReleases.Shipped.md' has a missing or invalid release header '# Shipped analyzer releases'